<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDoor</class>
 <widget class="QDialog" name="DialogDoor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>931</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="GB_DOOR">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>971</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬돋움</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox  {
    margin-top: 10px;
	font: 20pt &quot;함초롬돋움&quot;;
}

QGroupBox::title  {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 10px 8000px 10px 8000px;
    background-color: #595959;
    color: rgb(255, 255, 255);
}
QLabel {
	font: 10pt &quot;함초롬돋움&quot;;
}
QComboBox{
	font: 10pt &quot;함초롬돋움&quot;;
	background-color: #ffffff
}
QPushButton{
	font: 10pt &quot;함초롬돋움&quot;;
	background-color: #ffffff
}
QLineEdit{
	font: 10pt &quot;함초롬돋움&quot;;
	background-color: #ffffff
}



</string>
   </property>
   <property name="title">
    <string>출입구</string>
   </property>
   <widget class="QLabel" name="label_110">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>장치 연결 상태</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_DOOR_CONNECTION">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_DOOR_OPENED">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_111">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>문열림</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_DOOR_CLOSED">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_112">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬돋움</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>문닫힘</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>센서 A:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>센서 B:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LB_SENSOR_A">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>열림</string>
    </property>
   </widget>
   <widget class="QLabel" name="LB_SENSOR_B">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>190</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>열림</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
